src/main/java/wolox/training/controllers/UsersController.java

```diff
@@ -208,6 +210,32 @@ public void removeBook(@PathVariable Long id, @PathVariable Long bookid) {
     }
 
     /**
+     *Method to search user with birthdate beetween two date
```

Comment:
Can you add a space between * and the comment please?

Reply:
corrected